Models/Positions/Tile.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Chess_Backend.Models.Positions
+{
+    public class Tilecs
+    {
+    }
```

Comment:
Add two fields of type int which are x position and y position, then override Equals method and use the same field for the equals method